app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,70 @@
+class Api::V1::FactsController < ApplicationController
+    include AuthenticationCheck
+  
+    before_action :is_user_logged_in
+    before_action :set_fact, only: [:show, :update, :destroy]
+  
+    # GET /members/:member_id/facts
+    def index
+      @member = Member.find(params[:member_id])
+      render json: @member.facts # note that because the facts route is nested inside members
+                               # we return only the facts belonging to that member
+    end
+  
+    # GET /members/:member_id/facts/:id
+    def show
+      # your code goes here
+      if set_fact
+        render json: @fact, status: 200
+      else
+        render json: { error:
+        "The fact could not be found: #{@fact.error.full_messages.to_sentence}"},
+        status: 400
+      end
```

Comment:
Isn't this already handled by your ExceptionHandler? Additionally, `if set_fact` only passes when `@fact` exists, so you can't call `@fact.error.full_messages.to_sentence` in the `else` block.

While I'm here, the status code for "not found" should be 404, or in Rails you can use `:not_found`. If you like cats, you can use https://http.cat/ as a cheat sheet.